### DISCUSSION TITLE
Creating a parent with my own name?

### DISCUSSION THREAD

**Discussion by giantkiller** [piroor/treestyletab#3623] [General]
This addon says 'Tree' so how do I create a blank parent instead of a new tab that I cannot rename?

**Comment by irvinm** [comment]
What do you mean by (This addon says 'Tree')?

Are you talking about using group-tabs and renaming that tab as a parent of a tree?

- If you are, just create a group-tab and single click on the title to change the name.

![image](https://github.com/user-attachments/assets/aaef3fcf-0218-4b09-8eeb-dd93905b2b80)

**Reply by giantkiller** [reply]
I added a NEW TAB and cant rename it. This is what I cannot find.

**Reply by irvinm** [reply]
TST doesn't support renaming of tabs themselves.  (Or in this case the "title" of the tab)  There are other addons you can map different domains/URLs to have a specific title.  At the end of the day, TST is intended to mostly mimic what is on the native tab bar.

What is your use case here?  You can still use the group-tabs as above and rename it whatever you want to display on the tab.  What do you want to do/use after creating a new tab and changing the displayed title?

**Reply by giantkiller** [reply]
How do you start and rename a group tab? This is my stumbling block.

**Reply by irvinm** [reply]
Of course ... here are some ways to do it:

- Option 1 (Easiest):  Simply put "about:treestyletab-group?New%20Group" or "about:treestyletab-group?giantkillers%20new%20group%20tab" directly into the URL bar.  (If you just put "about:treestyletab-group" it will create a new tab with "Group" as the title which you can change.)
- Option 2:  Put "ext+treestyletab:group" into the URL bar ... click the "open link" button when prompted.
- Option 3:  
-- Go to about:addons and click on TST to bring up the options
-- At the top, check "Unlock Expert Options"
-- Go down to the "Development" section towards the bottom and click on the link named:  "Page for Group Tab"

Once the page is up, just single click on the name next to the folder on the page (not on the tab) and you should be able to edit the name and it will be reflected on the page and the tab title after you hit enter.

**Reply by giantkiller** [reply]
Option #1 is perfect. Thank you.

**Reply by giantkiller** [reply]
What about providing an option for tree tab background color? Like a color selection wheel of a choice or 16.

**Reply by irvinm** [reply]
You can make those changes yourself with TST CSS and I believe there are also some addons that deal with color you could use.

I like to add some color as well to differentiate my group tabs ... (yellow bars on the edges)

![image](https://github.com/user-attachments/assets/df6e9ee0-4a64-4953-9171-e0e7e982cbcd)

**Reply by teddymonster** [reply]
Hey this is an awesome feature that I was coming to try to recommend / find a way to do. I would love to see this as a mainline feature.

I would love to be able to add it to the list of options when spawning a new tab, "New Group Tab", with the option to rename it after creation, as well as assign color from the right click menu. Mostly for speed and simplicity. As a very squirrel brained individual, I have 1000s of tabs and I forget what some were related to

**Reply by piroor** [reply]
I have no plan to add the item to the "new tab" button due to the [project policy](https://github.com/piroor/treestyletab?tab=readme-ov-file#basic-project-policy). I recommend you to create a new bookmark to the bookmarks toolbar with the URL `about:treestyletab-group?New%20Group` manually.
